server/graphql/v2/query/collection/UpdatesCollectionQuery.ts

```diff
@@ -0,0 +1,47 @@
+import { GraphQLList, GraphQLNonNull, GraphQLString } from 'graphql';
+
+import models, { Op } from '../../../../models';
+import { UpdatesCollection } from '../../collection/UpdatesCollection';
+import { AccountReferenceInput, fetchAccountsIdsWithReference } from '../../input/AccountReferenceInput';
+import { CollectionArgs, CollectionReturnType } from '../../interface/Collection';
+
+const UpdatesCollectionQuery = {
+  type: new GraphQLNonNull(UpdatesCollection),
+  args: {
+    ...CollectionArgs,
+    tag: {
+      type: new GraphQLList(GraphQLString),
+      description: 'Only from accounts that have one of these tags',
+    },
+    host: {
+      type: new GraphQLList(AccountReferenceInput),
+      description: 'Host for the account for which to get updates',
+    },
+  },
+  async resolve(_: void, args): Promise<CollectionReturnType> {
+    const { offset, limit } = args;
+    let include;
+
+    if (args.host || args.tag) {
+      include = {
+        model: models.Collective,
+        as: 'collective',
+        required: true,
+        where: {},
+      };
+      if (args.host) {
+        const hostCollectiveIds = await fetchAccountsIdsWithReference(args.host);
+        include.where = { ...include.where, HostCollectiveId: hostCollectiveIds };
+      }
+      if (args.tag) {
+        include.where = { ...include.where, tags: { [Op.overlap]: args.tag } };
+      }
+    }
+
```

Comment:
Any condition that we need to add all the time? Making sure we're not exposing private or unpublished updates. How are we doing this in the collective updates resolver?

Reply:
@znarf Ah, thanks for catching this. This would have been bad. Added a condition to only get published updates!

Also added an `isPrivate: false` condition, even though the privacy applies to the html content only and is handled by the update resolver, it seems reasonable to not include private updates in this query. 

The collective updates resolver has an `onlyPublishedUpdates` (default: false) argument so that you as an admin can see the unpublished ones. But I'm thinking we don't need that in this query right now.